package.json

```diff
@@ -24,9 +24,9 @@
   "dependencies": {
     "@actions/core": "^1.10.1",
     "@actions/github": "^6.0.0",
-    "@octokit/core": "^5.1.0",
-    "@octokit/graphql-schema": "^14.57.0",
-    "@octokit/plugin-paginate-graphql": "^4.0.0",
+    "@octokit/core": "^6.1.1",
+    "@octokit/graphql-schema": "^15.4.1",
```

Comment:
Still blocker in https://github.com/kachick/wait-other-jobs/issues/713

https://github.com/octokit/graphql-schema/pull/924 waiting to be reviewed

Reply:
https://github.com/octokit/graphql-schema/releases/tag/v15.4.2

@dependabot recreate